platform/viewer/cypress/integration/pwa/OHIFExtensionMicroscopy.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('OHIF Microscopy Extension', () => {
+  before(() => {
+    cy.openStudyModality('SM');
+    cy.expectMinimumThumbnails(2);
+  });
+
+  it('checks if series thumbnails are being displayed', () => {
+    cy.get('[data-cy="thumbnail-list"]', { timeout: 15000 })
+      .contains('SM', { timeout: 15000 })
+      .its('length')
+      .should('to.be.at.least', 1);
+  });
+
+  it('drags and drop a SM thumbnail into viewport', () => {
+    cy.get('[data-cy="thumbnail-list"]', { timeout: 15000 })
+      .contains('SM', { timeout: 15000 })
+      .drag('.viewport-drop-target');
+
+    cy.get('.DicomMicroscopyViewer', { timeout: 15000 })
+      .its('length')
+      .should('be.eq', 1);
+
+    cy.wait(3000);
```

Comment:
I think this wait actually makes sense, but a small comment above it to indicate we're waiting for the "renderer" to display our image, and we're using this in lieu of something more exact/formal.